pyrsh.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import paramiko
+import argparse
+
+def Main():
+
+	# Описание скрипта
```

Comment:
@GitSergeyGit lets use EN lang for doc/comments, ok?

Reply:
+